terraform/service/main.tf

```diff
@@ -50,12 +50,24 @@ resource "aws_ecs_task_definition" "ecs_service_definition" {
           value = var.domain
```

Comment:
AWS_ACCOUNT_NAME is already being passed through to the container so you don't need AWS_ENVIRONMET. In you app code you should be able to just do an os.getenv("AWS_ACCOUNT_NAME") and that should be set to sdp-dev or whatever env you are deploying in to.